README.md

```diff
@@ -261,15 +261,8 @@ package managers:
 - `gem.txt`: run-time dependencies installed by gem.
 - `npm.txt`: run-time dependencies installed by npm.
 - `pip.txt`: a normal [pip `requirements.txt`][] file, for run-time
-  dependencies too.
-
-All of these files will install separately each line. See this example
```

Comment:
OK, but that reduces the possibility of putting specific switch for each pip line... We are losing them a feature that can be used. @lasley does this impact you?